projects/components/src/public-api.ts

```diff
@@ -186,6 +186,11 @@ export * from './select/select-option.component';
 export * from './select/select.component';
 export * from './select/select.module';
 
+// Splitter
+export * from './splitter/splitter';
```

Comment:
Do we have a use case for this?

Reply:
sorry meant to delete this when I saw it later :)

Reply:
No prob. Waterfall is a strong use case for this. Another use case is the upcoming geo map.  

It also think we can use it at few places where we use the two panel approach like explorer. Splitter would smooth out the interaction there.